pkgs/development/libraries/qt-5/5.12/default.nix

```diff
@@ -72,6 +72,10 @@ let
         # See https://github.com/NixOS/nixpkgs/issues/52457
         ./qtbase.patch.d/0014-qtbase-pkg-config.patch
 
+        # Patch framework detection to support X.framework/X.tbd,
+        # extending the current support for X.framework/X.
+        ./qtbase.patch.d/0015-qtbase-tbd-frameworks.patch
```

Comment:
This patch needs to be ported to every version of Qt in Nixpkgs.

Reply:
It applies trivially to the other version, so I've copied it over and added it to the patch lists. However neither qt514.qtbase nor qt515.qtbase currently build on master, so I can't test this.